src/cow-react/modules/wallet/web3-react/hooks/useWalletMetadata.ts

```diff
@@ -59,36 +60,11 @@ export function useWalletMetaData(): WalletMetaData {
     }
 
     if (connectionType === ConnectionType.GNOSIS_SAFE) {
-      return {
-        walletName: GNOSIS_SAFE_APP_NAME,
-        icon: SAFE_ICON_URL,
-      }
+      return METADATA_SAFE
     }
 
-    return { walletName: '', icon: '' }
```

Comment:
ATENTION: I'm a bit unsure why before this was returning empty strings. 

The type allows undefined. So I guess it should return that when is unknown.

Reply:
I think there are no special reasons, it should be undefined